Solutions/Azure Firewall/Analytic Rules/Azure Firewall - Multiple Sources Affected by the Same TI Destination.yaml

```diff
@@ -26,16 +26,16 @@ query: |
   let RunTime = 1d; 
   let StartRunTime = 1d; 
   let EndRunTime = StartRunTime - RunTime; 
-  let MinAffectedThershold = 5; 
+  let MinAffectedThreshold = 5;
   union isfuzzy=true
   (AzureDiagnostics 
   | where TimeGenerated  between (ago(StartRunTime) .. ago(EndRunTime))
   | where OperationName == "AzureFirewallThreatIntelLog"
   | parse msg_s with * "from " SourceIp ":" SourcePort:int " to " Fqdn ":" DestinationPort:int  "." * "Action: Deny. " ThreatDescription),
   (AZFWThreatIntel
   | where TimeGenerated between (ago(StartRunTime) .. ago(EndRunTime)))
-  | summarize TiTrafficCount = count(), dCountSourceIps = dcount(SourceIp), AffectedIps = make_set(SourceIp, 10000) by Fqdn, ThreatDescription,SourceIp 
```

Comment:
Hello @ricardotolentino, here SourceIp is removed. any specific reason for this.

Reply:
Hi @v-prasadboke.  Thanks for the feedback, you're right I missed that by removing the `SourceIp` aggregation then that affects its usage as an entity mapping.

How about instead of removing the entity mapping, I `mv-expand()` the `AffectedIps` array into a separate `SourceIp` column to restore the entity mapping?  The `MinAffectedThreshold` parameter would still work to filter the results.